src/app/containers/Metadata/index.jsx

```diff
@@ -39,33 +40,50 @@ const MetadataContainer = ({ metadata, promo }) => {
             themeColor,
             twitterCreator,
             twitterSite,
+            publishingPrinciples,
+            noBylinesPolicy,
           }) => {
             /* Canonical link generated from servicename and id */
             const canonicalLink = `https://www.bbc.com/${service}/articles/${id}`;
 
             return (
-              <Metadata
-                isAmp={platform === 'amp'}
-                articleAuthor={articleAuthor}
-                articleSection={metadata.passport.genre}
-                brandName={brandName}
-                canonicalLink={canonicalLink}
-                defaultImage={defaultImage}
-                defaultImageAltText={defaultImageAltText}
-                description={promo.summary}
-                facebookAdmin={100004154058350}
-                facebookAppID={1609039196070050}
-                lang={metadata.passport.language}
-                locale={locale}
-                metaTags={allTags(metadata.tags)}
-                themeColor={themeColor}
-                timeFirstPublished={timeFirstPublished}
-                timeLastPublished={timeLastPublished}
-                title={promo.headlines.seoHeadline}
-                twitterCreator={twitterCreator}
-                twitterSite={twitterSite}
-                type={metadata.type}
-              />
+              <Fragment>
+                <LinkedData
+                  isAmp={platform === 'amp'}
+                  lang={metadata.passport.language}
+                  type={metadata.type}
+                  seoHeadline={promo.headlines.seoHeadline}
+                  firstPublished={timeFirstPublished}
+                  lastUpdated={timeLastPublished}
+                  optimoId={id}
+                  service={metadata.createdBy}
+                  publishingPrinciples={publishingPrinciples}
+                  noBylinesPolicy={noBylinesPolicy}
+                  url={defaultImage}
```

Comment:
Why is this just `url`? Do you mean `logoUrl`?

Reply:
Because inside `LinkedData.jsx` we construct the JSON using the `linkMetadata` object, and the JSON+LD format requires a `url` property for `logo` and `image`.

I thought we were constructing the object with `{...props}` (or equivalent) and thus needed the same name. But looking at it again, I can refactor it out 👍

Reply:
Resolved by https://github.com/bbc/simorgh/pull/1391/commits/7d65aa5cda2271ea887c9235970839c2dc5809e5